llvm/utils/lit/tests/Inputs/shtest-glob/example_file1.txt

```diff
@@ -0,0 +1,2 @@
+## This is the first example file used for testing glob pattern matching.
+RUN: This is the first example file.
```

Comment:
```suggestion
```
This file should not contain any RUN: lines especially not ones that cannot actually be run. If you rename it to something like `example_file1.input` lit will not try to run it (since we have `config.suffixes = [".txt"]`).